ding/data/buffer/deque_buffer.py

```diff
@@ -209,6 +211,21 @@ def delete(self, indices: Union[str, Iterable[str]]) -> None:
         key_value_pairs = zip(remain_indices, range(len(indices)))
         self.indices = BufferIndex(self.storage.maxlen, key_value_pairs)
 
+    def save_data(self, file_name: str):
```

Comment:
maybe we can remove `export_data` and `import_data` methods in this class?

Reply:
fixed.